src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -399,12 +399,12 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
             <BidInfoRow
               label="Max bid"
               value={this.selectedBid().display}
-              onPress={() => this.goBackToSelectMaxBid()}
+              onPress={isLoading ? () => null : () => this.goBackToSelectMaxBid()}
```

Comment:
I think you want `isLoading ? null : () => this.goBackToSelectMaxBid() `, right? Right now, it's a function that returns null, which won't disable the button.

Reply:
This function is passed in to an `<TouchableWithoutFeedback>`, and a function that does nothing basically does the same as passing nothing from a user's perspective. I'm not that opinionated though. Thoughts?

Reply:
Oh, hmm I think I misunderstood. There's a bigger question I want to ask: should the rows and their buttons should have a visual "disabled" state while the bid is being placed? I think we should probably at least use a gray `color` for the button text (UI convention on iOS is that disabled buttons should be desaturated). Then we can use `null` as a disabled indicator. 

@briansw Can we get your input here?